spring-cloud-dataflow-module-deployers/spring-cloud-dataflow-module-deployer-marathon/src/main/java/org/springframework/cloud/dataflow/module/deployer/marathon/MarathonProperties.java

```diff
@@ -0,0 +1,96 @@
+/*
+ * Copyright 2015 the original author or authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.springframework.cloud.dataflow.module.deployer.marathon;
+
+import java.util.HashMap;
+import java.util.Map;
+
+import org.springframework.boot.context.properties.ConfigurationProperties;
+
+/**
+ * Configuration properties for connecting to a Marathon installation.
+ *
+ * @author Eric Bottard
+ */
+@ConfigurationProperties("marathon")
+public class MarathonProperties {
+
+	/**
+	 * The location of the Marathon REST endpoint.
+	 */
+	private String apiEndpoint = "http://10.141.141.10:8080";
+
+	/**
+	 * The docker image to use for launching modules.
+	 */
+	private String image = "springcloud/stream-module-launcher";
```

Comment:
Good idea. I would make it part of XD-3614 though, both not to disturb the merge of the 2 deployer impls (Marathon + k8s) and because there are other harmonizations to consider